generated/niscope/niscope.proto

```diff
@@ -1090,16 +1090,16 @@ message ConfigureChanCharacteristicsResponse {
 message ConfigureClockRequest {
```

Comment:
I know I requested it, but looking at this one and the way things get sorted below, I'm wondering whether there's really any value at all in this PR.

Reply:
My two cents: I like it because it makes most of the messages with oneof's marginally saner. Also, with this one at least within each oneof the fields are sorted by number now.

I'm fine not doing it but I think it's a little better than it was and having it in the proto gen will fix up future oneof's as well.

Reply:
Thanks Ryan. Sorry to scare you Illya. When the output isn't exactly what one envisions, it's worth questioning.

Reply:
@astarche , does this look good to you? Not sure if you saw Dan's comment on one of the other PRs that led to this change.

Reply:
Yeah, I like it. We can have non-sequential fields in oneofs either (a) as a compatibility preserving change (add a new `oneof` to existing message) or (b) as a mistake (? maybe it's semi-intentional, but seems like we'd be better off without it). Not sure which one this is. But, I think either way it makes more sense if they're as much in order as they can be.